daphne_server/docker-compose-e2e.yaml

```diff
@@ -46,6 +48,9 @@ services:
       target: helper
     depends_on:
       - helper_storage
+    environment:
+      - DAP_SERVICE__SIGNING_KEY=${DAP_SIGNING_KEY}
```

Comment:
Why use a different variable name here? Why not just go with `DAP_SIGNING_KEY`? Also, what's with the double underscore?

Reply:
This is a consequence of the [`config`](https://docs.rs/config/latest/config/struct.Environment.html) crate.

`DAP_` is the [env prefix](https://github.com/cloudflare/daphne/blob/6e1d98b70244ba14d6bf8eedaf384a0644651eef/daphne_server/examples/service.rs#L39)
`__` is [equivalent to the `.` operator in rust](https://github.com/cloudflare/daphne/blob/6e1d98b70244ba14d6bf8eedaf384a0644651eef/daphne_server/examples/service.rs#L41)

Hence, `DAP_SERVICE__SIGNING_KEY` maps to `config.`[`service`](https://github.com/cloudflare/daphne/blob/mendess/upstream-hpke-config-signing/daphne_server/examples/service.rs#L17)`.`[`signing_key`](https://github.com/cloudflare/daphne/blob/mendess/upstream-hpke-config-signing/daphne_service_utils/src/config.rs#L96)